### ISSUE TITLE
Auto state entry doesn't appear under the preview features

### ISSUE THREAD

**Issue by SteeveDess** [joachimdalen/azdevops-auto-state#35]
Hi,

I installed auto state extension from the market place. 
I'm admin of My azure organization. It's a free version for testing purpose and training.

I did find the preview features' menu. But I can't locate the auto state line to activate it.
Any idea why ?

For your information, New boards hubs is off.
Regards
Steeve.

**Comment by joachimdalen** [comment]
Hi Steeve!

The only thing I can think of is the feature scope. Auto State is scoped at a project level which means you need to be inside a project for the feature flag to appear. (The url should look something like `https://dev.azure.com/ORGANIZATION/PROJECT`)

When you open the modal with all the feature flags the dropdown should have three options

- `for me [Your name]`
- `for this project [Project Name]` (This is the one you should select)
- `for this organization [Organization Name]`

Could you please check this and see if the toggle appear? I'll also clarify this in the documentation.

**Comment by SteeveDess** [comment]
Hi Joachim,

Thank you for your very quick answer.
It's exactly the point. 

I was able to activate auto state.
And the Extensions Entry appeared in my project settings.
It was disturbing because this entry was only visible at the organization level before I activated auto state. So It was at this level I was looking, and not at the project level.
Thank you for your time.

Have a nice weekend.
Regards
Steeve
